packages/react-hook-form-macaw/.storybook/preview.tsx

```diff
@@ -0,0 +1,82 @@
+import "@saleor/macaw-ui/next/style";
+import "./styles.css";
+
+import React from "react";
+import { DocsContainer, DocsContainerProps } from "@storybook/blocks";
+import { PropsWithChildren } from "react";
+import { Box, ThemeProvider, useTheme } from "@saleor/macaw-ui/next";
+
+type MacawDocsContainerProps = {
+  [K in keyof DocsContainerProps]: K extends "context"
+    ? DocsContainerProps[K] & { store: Record<string, any> }
+    : DocsContainerProps[K];
+};
+
+const MacawDocsContainer = ({ children, ...props }: PropsWithChildren<MacawDocsContainerProps>) => {
+  return (
+    <ThemeProvider defaultTheme={props.context.store.globals.globals.theme}>
+      <ThemeSwitcher theme={props.context.store.globals.globals.theme}>
+        <DocsContainer {...props}>{children}</DocsContainer>
+      </ThemeSwitcher>
+    </ThemeProvider>
+  );
+};
+
+export const parameters = {
```

Comment:
please make these values to `satisfies` or have assigned types, so we can safely edit them later, I assume storybook has typings

Reply:
Improved types as much as I could. Fell into the rabbithole of SB generics and there are known issues with it.